templates/vue/src/components/NodeModal.vue

```diff
@@ -212,11 +221,20 @@ export default {
         ? true
         : wpData.wpCanEditTapestry !== ""
     },
+    accessSubmit() {
+      if (!this.ready || this.fileUploading) {
+        return false
+      }
+      return this.node.mediaType === "video" || this.node.mediaType === "h5p"
+    },
```

Comment:
When I open the modal, the submit button is inaccessible by default; this computed property is returning false.